include/mme-app/contextManager/mmeErabModIndProcedureCtxtManager.h

```diff
@@ -0,0 +1,63 @@
+ /*
+ * Copyright 2019-present, Infosys Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
small copyright.

Reply:
This requires script. Will do it as separate activity.